README.markdown

```diff
@@ -1,3 +1,5 @@
+![alt text](logo.png "watchman")
```

Comment:
The logo already exists in the repo under `website/static/logo.png`; could you update this to reference that path and then remove the added `logo.png`?